virttest/utils_libvirt/libvirt_misc.py

```diff
@@ -0,0 +1,43 @@
+"""
+libvirt related misc functions
+"""
+import logging
+
+from avocado.core import exceptions
+
+from virttest import libvirt_version
+
+
+def is_libvirt_feature_supported(params, ignore_error=False):
```

Comment:
it only use one parameter :func_supported_since_libvirt_ver, do we really need pass in whole dict?

Reply:
I'm using a dict type argument because:
1. the values can be set in cfg
2. easy to extend the arguments